contrib/nightly/plugins.yml

```diff
@@ -6,3 +6,11 @@
   description: |
     Tutor plugin to enable execution of untrusted code in secure sandboxes.
     In particular, this plugin is required to run Python-evaluated input XBlocks.
+- name: pod-autoscaling
```

Comment:
Add an empty line just above (and in the olive file as well).

Reply:
@regisb addressed